.github/workflows/pull_requests.yaml

```diff
@@ -303,7 +302,6 @@ jobs:
           go-version: '1.21'
           cache: true
       - name: Acceptance tests Large (modules)
-        uses: nick-fields/retry@v3
```

Comment:
**note (if-minor):** is this modules ?

Reply:
```
    name: acceptance-tests-large
    runs-on: ubuntu-latest-4-cores
    strategy:
      fail-fast: false
      matrix:
        test: ["--acceptance-go-client-named-vectors"]
    steps:
      - uses: actions/checkout@v4
      - name: Set up Go
        uses: actions/setup-go@v5
        with:
          go-version: '1.21'
          cache: true
      - name: Acceptance tests Large (modules)
        uses: nick-fields/retry@v3
        with:
          # 15 Minute is a large enough timeout for most of our tests
          timeout_minutes: 15
          max_attempts: 3
          command: ./test/run.sh ${{ matrix.test }}
          on_retry_command: ./test/run.sh --cleanup
```

I think the name is wrong - will quickly remove the `modules`